portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,35 +27,38 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> funFacts = new ArrayList<String>();
+  private ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Change this to List<String> comment = ..

List is the interface & ArrayList is an implementation of List. If you use List, at a later time you can change the implementation to use SortedList or something and not impact any of the code.

Reply:
Done